src/state/stake/hooks.ts

```diff
@@ -539,7 +538,7 @@ export function useOldSyrupInfo(
           memo.push({
             stakingRewardAddress: rewardsAddress,
             token: syrupInfo.token,
-            ended: syrupInfo.ended,
```

Comment:
Do we need that field in the JSON metadata anymore?

Reply:
I think we do not need ended field for syrup